packages/services/src/services/Locale/Locale.js

```diff
@@ -118,8 +117,6 @@ class LocaleAPI {
         const codes = lang.split('-');
         return { cc: codes[1], lc: codes[0] };
       }
-    } else {
-      return _localeDefault;
```

Comment:
@kennylam this should be added back if the lang attribute is not set, no? the `getLang` method is being used in other places as well (e.g. setting the `lc/cc` for the Masthead SearchTypeahead API)

cc: @annawen1

Reply:
@jeffchew @annawen1  Returning a default language there will result in the first condition as always true. Cookies will never be get/set from the second condition.
https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/services/src/services/Locale/Locale.js#L70

I'll need to take a look and see how it affects MastheadSearch

Reply:
Cleared this up for @annawen1 and she explained that the second condition is sort of old functionality that is being left there for the time being. Will revert to default locale